iOS/AnimatedShapeButton/source/AnimatedShapeButton/AnimatedShapeButton.designer.cs

```diff
@@ -0,0 +1,20 @@
+// WARNING
+//
+// This file has been generated automatically by Visual Studio from the outlets and
+// actions declared in your storyboard file.
+// Manual changes to this file will not be maintained.
+//
+using Foundation;
+using System;
+using System.CodeDom.Compiler;
+
+namespace AnimatedButtons
+{
+    [Register ("AnimatedShapeButton")]

```

Comment:
I think this file is not needed - it gets generated incorrectly.

Reply:
Thanks @mattleibow, I removed it.